### PR TITLE
New Login and Date Feature

### PR DESCRIPTION
**Pull request by kumarharmesh**
Hi Sir

I have forked your repo and added two new utitlty classes. I have tested it also. Please accept pull request after review.
Please let me know if you have any doubts or you find any incompatibility

kumarharmesh@gmail.com

### CODE REVIEW

**Review by HarmeshKr — COMMENTED** [review]
Please make Class1 public

**Comment by kumarharmesh** [comment]
Hello sir, as per your suggestion I have made it public

**Review by HarmeshKr — CHANGES_REQUESTED** [review]
Please make GetDate method return Datetime object

**Comment by kumarharmesh** [comment]
Ok Sir. I have made the required changes . DateTime is now returned

File: ConsoleAppDay5/Class2.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleAppDay5
+{
+    internal class Class2
+    {
+        public class Login
+        {
+            public event Action<string> LoginEvent;
```
Comment (HarmeshKr) [inline ConsoleAppDay5/Class2.cs:13]: Please make sure action has two string params for uid and pwd

**Review by HarmeshKr — APPROVED** [review]
Now all correct @kumarharmesh #1

**Pull request merged**